src/crypto.cc

```diff
@@ -136,8 +147,8 @@ bool Crypto::VerifySignature(const PublicKey &public_key, const std::string &sig
   }
 }
 
-bool Crypto::parseP12(FILE *p12_fp, const std::string &p12_password, const std::string &pkey_pem,
-                      const std::string &client_pem, const std::string ca_pem) {
+bool Crypto::parseP12(FILE *p12_fp, const std::string &p12_password, std::string *out_pkey, std::string *out_cert,
```

Comment:
This is a Google Style Guide thing https://google.github.io/styleguide/cppguide.html#Reference_Arguments